index.js

```diff
@@ -38,18 +65,32 @@
  * And returns the first and last name concatenated together (i.e. 'Bob Evans').
  */
 
+function getFullName (object) {
+	return object.firstName + ' ' + object.lastName;
+};
 
 /**
  * Q6: Write a function called `reverseString` that has a String parameter and returns
  * the String reversed. Hint: look up `.split()`
  */
-
+function reverseString(string) {
+	var backwards = string.split('').reverse().join('');
+	return backwards;
+}
```

Comment:
Once again, this solution is perfect. If I were writing this function, I might just do it all in one line instead of using an intermediate variable:

```js
function reverseString(string) {
   return string.split('').reverse().join('');
}
```